apps/cms.c

```diff
@@ -721,15 +721,15 @@ int cms_main(int argc, char **argv)
         goto end;
 
     if (digestname != NULL) {
-        if (!opt_md(digestname, &sign_md))
```

Comment:
A smarter thing to do would have been to just put the fetch inside opt_md

Reply:
I got rid of this change.  Too big...